cstore_reader.c

```diff
@@ -684,6 +699,13 @@ LoadStripeSkipList(FILE *tableFile, StripeMetadata *stripeMetadata,
 		ColumnBlockSkipNode *columnSkipList = NULL;
 		uint32 blockIndex = 0;
 
+		/* no need to create ColumnBlockSkipList if the column is not selected */
+		if (!projectedColumnMask[columnIndex] && columnIndex > 0)
```

Comment:
We still need to make firstColumn check even if we do that.
replaced ```columnIndex == 0```  and ```columnIndex > 0``` checks with equivalent ```firstColumn``` expressions